images/retag-and-push/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Retag and push images
+description: |
+  This action will, as stated in its name, retag and push images
+
+inputs:
+  src-image:
+    description: |
+      The image that will be retagged
+    required: true
+  dst-image:
+    description: |
+      A target image that will be generated by retagging src-image and later
+      pushed
+    required: true
+  username:
+    description: |
+      Username used to login to the repository the push is aimed at
+    required: true
+  password:
+    description: |
+      Password used to login to the repository the push is aimed at
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: docker/login-action@v2
```

Comment:
There's a pitfall here: if source or destination images are not in Quay, but require authentication, how is that solved? This action only performs a login to Quay.
It might be better to require the users of the action to perform necessary logins before this action, like you did in https://github.com/stackrox/berserker/pull/4/files#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3R34-R39.

Reply:
The registry is now configurable and the default is quay.io. The username and password are now optional and login is only done if those are provided.

Reply:
The registry is now determined from the destination image.

Reply:
I understand that for the current use cases it is fine to not require a login to the source registry.
Could you add that caveat to the action README?